Content.Server/GameObjects/EntitySystems/ConveyorSystem.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System.Collections.Generic;
+using Content.Server.GameObjects.Components.Conveyor;
+using Content.Shared.GameObjects.Components.Conveyor;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    [UsedImplicitly]
+    public class ConveyorSystem : EntitySystem
+    {
+        private uint _nextId;
+        private readonly Dictionary<uint, ConveyorGroup> _groups = new Dictionary<uint, ConveyorGroup>();
```

Comment:
I don't see anything here to get rid of old conveyor groups that have nothing in them anymore.